Adyen/Core/Models/DisplayInformation.swift

```diff
@@ -44,18 +56,37 @@ public struct DisplayInformation: Equatable {
     /// - Parameter footnoteText: The footnote text if any.
     /// - Parameter accessibilityLabel: An optional custom `accessibilityLabel` to use.
     /// Set this if the title / subtitle might not be sufficient enough to provide a good accessibility
+    @available(*, deprecated, renamed: "init(title:subtitle:logoName:trailingInfo:footnoteText:accessibilityLabel:)")
```

Comment:
Not sure about this deprecation. What happens if they use the old deprecated method and set something for the disclosure text property which won't exist anymore?

Reply:
the `disclosureText` was `@_spi(AdyenInternal)` so there is no other publicly noticable change than the deprecation

Reply:
The proerty yes, but I meant what happens if it's set on init, expecting to see some disclosure text but nothing will be visible?

Reply:
It will be `trailingInfo: disclosureText.map { .text($0) },`